ci-operator/config/stolostron/acm-config/stolostron-acm-config-main.yaml

```
@@ -17,7 +17,7 @@ tests:
   steps:
     env:
       DESTINATION_VERSIONS: 5.0 5.1
-      LAST_RELEASE_VERSION: "2.17"
+      DESTINATION_VERSIONS: 5.1 5.2
```

YAMLlint (1.37.1)

[error] duplication of key "DESTINATION_VERSIONS" in mapping

(key-duplicates)